README.md

```diff
@@ -36,6 +36,14 @@ Or for Gradle, add to your dependencies:
 compile 'io.grpc:grpc-all:0.9.0'
 ```
 
+For Android client, you only need to depend on the needed sub-projects, such as:
+```gradle
+compile 'io.grpc:grpc-core:0.9.0'
```

Comment:
Core should get pulled in via stub. Do we want to list it explicitly, or make this list minimal?